### PR TITLE
[BACK-1170] Marketo retries

### PR DESCRIPTION
**Pull request by HaroldBernard**
Added retries to marketo to retest connection on initial fail on Shoreline start

### CODE REVIEW

File: user/api_test.go [hunk 1]
```diff
@@ -66,33 +77,61 @@ var (
...
 }
 
 ////////////////////////////////////////////////////////////////////////////////
+// LIKE RECTANGLE what is needed in Area and Perimeter
+type MockUser struct {
+}
+type MockManager struct {
+}
+
+func (U *MockUser) Email() string {
```
Comment (derrickburns) [inline user/api_test.go]: Do you use this?

Comment (HaroldBernard) [reply user/api_test.go]: I dont believe so since it is not needed to create the manager.

**Comment by HaroldBernard** [comment]
Back-1170

File: user/api_test.go [hunk 2]
```diff
@@ -51,6 +54,14 @@ var (
 		Salt:               "a mineral substance composed primarily of sodium chloride",
 		VerificationSecret: "",
 		ClinicDemoUserID:   "00000000",
+		Marketo: marketo.Config{
```
Comment (derrickburns) [inline user/api_test.go]: Is this used?

Comment (HaroldBernard) [reply user/api_test.go]: in the ApiConfig struct it has a marketo key, so I added those fake marketo keys and values to the  fake apiconfig

**Comment by HaroldBernard** [comment]
@tjotala I tested this code locally and on QA1. Let me know if it is ok to merge.

**Pull request merged**